Sky Atlas/Sky Atlas.html

```diff
@@ -24754,21 +24831,29 @@ <h1>Agent Scope Database</h1>
           <td>Core</td>
           <td>At the conclusion of the poll, if the proposal is approved, the Operational Facilitator submits the edit to Powerhouse to formally update the Agent Artifact. Regardless of the outcome, the Operational Facilitator updates the Powerhouse System to include the result of the vote, including any pertinent documents.</td>
         </tr>
+        <tr>
+          <td>
+            <dfn>Short Term Transitionary Measures</dfn>
+          </td>
+          <td>Spark</td>
+          <td>Core</td>
+          <td><p>Until the Powerhouse system supports updating Agent Artifacts, the Operational Facilitator works with the Governance Facilitators to update the Atlas GitHub repository located at <a href="https://github.com/makerdao/next-gen-atlas/pulls">Pull requests · makerdao/next-gen-atlas · GitHub</a> to reflect proposals approved by Star Governance.</p></td>
```

Comment:
nit: You don't need p tags for a single line entry

```suggestion
          <td>Until the Powerhouse system supports updating Agent Artifacts, the Operational Facilitator works with the Governance Facilitators to update the Atlas GitHub repository located at <a href="https://github.com/makerdao/next-gen-atlas/pulls">Pull requests · makerdao/next-gen-atlas · GitHub</a> to reflect proposals approved by Star Governance.</td>
```